examples/MagicWeather/app/src/main/java/com/revenuecat/sample/MainApplication.kt

```diff
@@ -22,7 +22,7 @@ class MainApplication : Application() {
         Initialize the RevenueCat Purchases SDK.
 
         - appUserID is nil, so an anonymous ID will be generated automatically by the Purchases SDK. Read more about Identifying Users here: https://docs.revenuecat.com/docs/user-ids
-        - observerMode is false, so Purchases will automatically handle finishing transactions. Read more about Observer Mode here: https://docs.revenuecat.com/docs/observer-mode
+        - finishTransactions is true, so Purchases will automatically handle finishing transactions. Read more about finishing transactions here: https://docs.revenuecat.com/docs/finishing-transactions
```

Comment:
**Note:** `/finishing-transactions` is not live yet. See https://github.com/RevenueCat/docusaurus/pull/261